apps/omg_watcher/lib/exit_processor.ex

```diff
@@ -361,12 +359,13 @@ defmodule OMG.Watcher.ExitProcessor do
     {:reply, response, state}
   end
 
-  def handle_call({:create_challenge, Utxo.position(blknum, txindex, oindex) = exiting_utxo_pos}, _from, state) do
+  def handle_call({:create_challenge, Utxo.position(blknum, _txindex, oindex) = exiting_utxo_pos}, _from, state) do
     with spending_blknum_response <- exiting_utxo_pos |> Utxo.Position.to_db_key() |> OMG.DB.spent_blknum(),
-         %{txhash: txhash} <- OMG.Watcher.DB.Transaction.get_by_position(blknum, txindex),
-         {:ok, exit_id} <- OMG.Eth.RootChain.get_standard_exit_id(txhash, oindex),
-         {:ok, raw_spending_proof, exit_info} <-
-           Core.get_challenge_data(spending_blknum_response, exiting_utxo_pos, state) do
+         {:ok, {block_hash, _block_timestamp}} <- Eth.RootChain.get_child_chain(blknum),
+         {:ok, [block]} <- OMG.DB.blocks([block_hash]),
+         {:ok, raw_spending_proof, %ExitInfo{exit_txhash: exit_txhash} = exit_info} <-
```

Comment:
return `exit_txhash` separately, not inside the `ExitInfo` struct: `{:ok, raw_spending_proof, exit_info, exit_txhash} <- ...`. No reason to leverage `ExitInfo` here, which is used totally differnet by the `ExitProcessor`